Sprint 1/step 1

pvl1001 opened the pull request.

Commit:
Верстка AppHeader

Commit:
Верстка BurgerIngredients

Commit:
Верстка BurgerConstructor

Commit:
Добавить PropTypes

andryxxa93 commented on src/components/AppHeader/AppHeader.js:
```diff
@@ -0,0 +1,55 @@
+function AppHeader() {
+   const type = ( isActive ) => isActive ? 'primary' : 'secondary'
+
+   const [ navBtns ] = useState( () => [
```
**Можно лучше**
в шапке активный пункт меню будет определяться с помощью роутинга, это будет реализовано в 3 спринте, не нужно сейчас это делать через стейт

andryxxa93 commented on src/components/AppHeader/LinkHeader/LinkHeader.js:
```diff
@@ -0,0 +1,16 @@
+import s from './LinkHeader.module.scss'
+
+
+function LinkHeader( { icon, isActive, name, className = '' } ) {
```
**Необходимо исправить**
У всех компонентов, которые принимают пропсы должна быть типизация через `propTypes`.

andryxxa93 reviewed (changes requested):
Отличная работа! Проект соответствует макету, код написан чисто и аккуратно. Очень понравилось, что вы правильно типизировали ингредиент и использовали `isRequired`.

Есть одно замечание, которое мы сейчас можете не исправлять, но на финальном этапе сдачи работы это станет обязательно.
- Необходимо чтобы все компоненты, которые принимают пропсы имели типизацию через `propTypes`.

В pull-request вы так же найдете рекомендации **как можно улучшить** код, их делать не обязательно, но в будущем они помогут избежать ошибок и сделать код лучше.

Рекомендация: для повышения читабельности кода и избежания ошибок рекомендую добавить в проект `eslint` и `pretier.` Они помогут в этом. https://dev.to/knowankit/setup-eslint-and-prettier-in-react-app-357b

Удачных правок!

Commit:
fix / BurgerConstructor

- Разделить массив данных на булки и начинку
- зафиксировать положение булок
- поправить верстку

Commit:
fix LinkHeader / добавить PropTypes
